config/locales/en.yml

```diff
@@ -543,6 +543,9 @@ funding:
         reset: Reset filters
         clearSelection: Clear selection
         toggle: Toggle
+        organisationLegend: Organisation
```

Comment:
This separates out the FieldGroup names from the individual fields in those groups.

Reply:
👍